documentation/asciidoc/computers/configuration/raspi-config.adoc

```diff
@@ -217,7 +210,7 @@ Configure the network's proxy settings.
 
 ==== Boot order
 
-On Raspberry Pi 4 and later, specify whether to boot from USB or network if the SD card isn't inserted. For more information, see xref:raspberry-pi.adoc#raspberry-pi-bootloader-configuration[bootloader configuration].
+On Raspberry Pi 4 and later, specify whether to boot from USB or network in absence of an SD card. For more information, see xref:raspberry-pi.adoc#raspberry-pi-bootloader-configuration[bootloader configuration].
```

Comment:
Or SSD, on a Pi 5 with an M.2 HAT?

Reply:
I dunno if that's the right wording, as USB is also a "valid boot media" :thinking: